ofnet.go

```diff
@@ -287,3 +289,21 @@ type PortInfo struct {
 	MbrLinks    []*LinkInfo
 	ActiveLinks []*LinkInfo
 }
+
+// PortUpdates maintains multiplae port update info
+type PortUpdates struct {
+	PortName string
+	Updates  []PortUpdate `json:"updates,overflow"`
+}
+
+// PortUpdate maintains information about port update
+type PortUpdate struct {
+	UpdateType string      `json:"update-type,omitinfo"`
```

Comment:
IMHO, having an update API is a cleaner way to handle this and any future updates. Currently we are only handling LACP update in this PR.